tidb-in-kubernetes/upgrade-tidb.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
This file should be put in the docs-cn/how-to/upgrade/ directory. You can refer to [TOC with TiDB in Kubernetes Integrated](https://docs.google.com/document/d/1k7zvGbcoMj1ggo28uZ5gr3-GXNiPAIWda7Df1oXOZbM/edit?usp=sharing) for documentation structure.